smoke-test/tests/cypress/cypress/e2e/searchV2/searchBarV2Redesigned.js

```diff
@@ -0,0 +1,232 @@
+import { applyGraphqlInterceptors, getThemeV2Interceptor } from "../utils";
+import { SearchV2Helper } from "./helpers/searchV2Helper";
+
+const SAMPLE_ENTITY_NAME = "SampleCypressKafkaDataset";
+const SAMPLE_ENTITY_URN =
+  "urn:li:dataset:(urn:li:dataPlatform:kafka,SampleCypressKafkaDataset,PROD)";
+
+describe("searchBarV2", () => {
+  const helper = new SearchV2Helper();
+
+  const setupTest = (searchBarApi = "AUTOCOMPLETE_FOR_MULTIPLE") => {
+    cy.on(
+      "uncaught:exception",
+      (err) =>
+        !err.message.includes(
+          "ResizeObserver loop completed with undelivered notifications",
+        ),
+    );
+
+    applyGraphqlInterceptors([
+      getThemeV2Interceptor(true),
+      helper.getSearchBarInterceptor(true, searchBarApi),
+    ]);
+
+    cy.loginWithCredentials();
+    cy.skipIntroducePage();
+  };
+
+  it("can handle all entities search", () => {
+    setupTest();
+    helper.goToHomePage();
+
+    helper.searchBarV2.type("*{enter}");
+    helper.waitForSearchPageLoading();
+
+    helper.ensureHasResults();
```

Comment:
yeah like we might not even need the wait here since there's a 10 second timeout by default to get what we're looking for